conftest.py

```diff
@@ -68,15 +68,19 @@ def odin_postgresql(docker_ip, docker_services):
 
 
 @pytest.fixture(scope='session')
-def odinapi_service(docker_ip, docker_services):
+def odinapi_service(docker_ip, docker_services, docker_compose_file):
+    print(docker_compose_file)
     port = docker_services.port_for('odin', 80)
     url = "http://{}:{}".format(docker_ip, port)
     docker_services.wait_until_responsive(
         timeout=WAIT_FOR_SERVICE_TIME,
         pause=PAUSE_TIME,
         check=lambda: odinapi_is_responsive(url),
     )
-    return url
+    yield url
+    logs = docker_services._docker_compose.execute('logs webapi')
+    for line in logs.decode().split('\n'):
+        print(line)
```

Comment:
debug? please remove

Reply:
No not debug, when dockerized tests fail we don't get any info from what happened in the container, that print gives use that and it is silent in the results until a test fails, because that is how tox works.